chart/requirements.yaml

```diff
@@ -14,9 +14,8 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
 dependencies:
   - name: postgresql
-    version: 6.3.12
-    repository: "@stable"
+    version: 9.1.2
+    repository: "https://charts.bitnami.com/bitnami"
```

Comment:
You made the same changes in two PRs (another is https://github.com/apache/airflow/pull/10598).

As shared, more clarifications may be needed to justify the changes ( at least to me). Meanwhile, please clean-up the two PRs.